ambry-test-utils/src/main/java/com/github/ambry/account/InMemAccountService.java

```diff
@@ -167,7 +167,11 @@ public synchronized void clear() {
    */
   public synchronized Account createAndAddRandomAccount() {
     Account account = generateRandomAccount();
-    updateAccounts(Collections.singletonList(account));
+    try {
+      updateAccounts(Collections.singletonList(account));
+    } catch (AccountServiceException ase) {
+      throw new IllegalStateException(ase);
```

Comment:
Is this because  throwing AccountServiceException will bring lots of changes in too many places?

Reply:
Pretty much.